bedrock/firefox/templates/firefox/firstrun/firstrun-quantum.html

```diff
@@ -38,45 +38,54 @@
-        {% endif %}
-        </p>
-        <p><a href="{{ url('firefox.accounts') }}">{{_('Learn more about Firefox Accounts')}}</a></p>
-      {% endblock %}
-      </header>
-      <div id="fxaccounts-wrapper">
-        {{ fxa_email_form(
-          entrypoint='mozilla.org-firstrun',
-          utm_source='firstrun',
-          button_class='mzp-c-button mzp-t-product')
-        }}
+{% block page_image %}{{ static('img/firefox/template/page-image-master.jpg') }}{% endblock %}
 
-        <p class="fxa-signin">{{ _('Already have an account?') }} <a {{ fxa_link_fragment(entrypoint='mozilla.org-firstrun_page', utm_params={'campaign': 'fxa-embedded-form', 'medium': 'referral', 'content': 'firstrun-page', 'source': 'firstrun-page'}) }} class="js-fxa-cta-link">{{ _('Sign In') }}</a></p>
+{% block content %}
+  <section class="hero mzp-has-image mzp-t-firefox mzp-t-product-firefox">
+    <div class="page-content">
+      
+      <div class="mzp-l-content">
+        <div class="mzp-c-hero-body">
+          <div class="browser-logo">
+            <img class="browser-logo" src="{{ static('img/logos/firefox/logo-quantum-full-wordmark-high-res.png') }}" />
+          </div>
+          <h1 class="mzp-c-hero-title">
+            {% if l10n_has_tag('firstrun_email_only_form') %}
+              {{ _('Take Firefox with You') }}
+            {% else %}
+              {{_('Already using Firefox?')}}
+            {% endif %}
+          </h1>
+          <div class="mzp-c-hero-desc">
+            <p>
+            {% if l10n_has_tag('firstrun_email_only_form') %}
+              {{ _('Get your bookmarks, history, passwords and other settings on all your devices.') }}
+            {% else %}
+              {{_('Sign in to your account and we’ll sync the bookmarks, passwords and other great things you’ve saved to Firefox on other devices.')}}
+            {% endif %}</p>
+          </div>
+          <aside class="mzp-c-box-emphasis" >
+            <div class="mzp-c-box-emphasis-content" id="fxaccounts-wrapper">
+              {{ fxa_email_form(
+                entrypoint='mozilla.org-firstrun',
+                utm_source='firstrun',
+                button_class='mzp-c-button mzp-t-product')
+              }}
+              <p class="fxa-signin">{{ _('Already have an account?') }} <a {{ fxa_link_fragment(entrypoint='mozilla.org-firstrun_page', utm_params={'campaign': 'fxa-embedded-form', 'medium': 'referral', 'content': 'firstrun-page', 'source': 'firstrun-page'}) }} class="js-fxa-cta-link">{{ _('Sign In') }}</a></p>
+            </div>
+          </aside>
+        </div>
       </div>
-    </main>
-
-  </div>{# /#scene #}
+      <div class="mzp-c-hero-image">
+        <img class="hero-photo" src="{{ static('img/firefox/firstrun/deviceSwoosh.gif') }}" alt="gif showing three devices with a swooshing color motion going across all screens">
```

Comment:
I don't know if this image is meaningful enough to warrant alt text, it seems mostly decorative. But if we do want to provide alt text it should be enclosed as a string for translation.